api/v1beta1/nutanix_types.go

```diff
@@ -35,6 +38,12 @@ const (
 	// NutanixBootTypeUEFI is a resource identifier identifying the UEFI boot type for virtual machines.
 	NutanixBootTypeUEFI NutanixBootType = "uefi"
 
+	// NutanixGPUIdentifierName is a resource identifier identifying a GPU by Name.
+	NutanixGPUIdentifierName NutanixGPUIdentifierType = "name"
+
+	// NutanixGPUIdentifierDeviceID is a resource identifier identifying a GPU using device ID.
+	NutanixGPUIdentifierDeviceID NutanixGPUIdentifierType = "deviceId"
```

Comment:
nit: consider using deviceID or device_id; deviceId seems funky (although not for any particular reason, hence a nit).

Reply:
renaming to deviceID